.kitchen.yml

```diff
@@ -36,7 +37,7 @@ default: &default
 suites:
   - name: default
```

Comment:
Since Patrick is out I will probably merge this as it is and discuss the matrix when he is back.